config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
```

Comment:
It used to be `files`, right? Will we run into any problems because of that change for existing instances (expected symlinks or the like)?